hooks/useGovernanceAssets.ts

```diff
@@ -680,7 +679,34 @@ export default function useGovernanceAssets() {
       name: 'Token Register Trustless',
       packageId: PackageEnum.MangoMarketV4,
     },
-
+    [Instructions.MangoV4GroupEdit]: {
+      name: 'Group Edit',
+      packageId: PackageEnum.MangoMarketV4,
+    },
+    [Instructions.MangoV4Serum3EditMarket]: {
+      name: 'Serum 3 Edit Market',
```

Comment:
in UI we want to rename all serum3 to Openbook, we are not renaming the ixs on the program because its a lot of work, and openbook and serum program code is identical

Reply:
Rename to `Edit Opebook Market`